modules/810-deckhouse-web/templates/pdb.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  annotations:
+    helm.sh/hook: post-upgrade, post-install
+    helm.sh/hook-delete-policy: before-hook-creation
+  name: deckhouse-web
+  namespace: d8-system
+{{ include "helm_lib_module_labels" (list . (dict "app" "deckhouse-web")) | indent 2 }}
```

Comment:
As I remember we use `nindent` everywhere

Reply:
done